core/src/labels/label.cpp

```diff
@@ -132,8 +131,11 @@ void Label::enterState(State _state, float _alpha) {
 }
 
 void Label::setAlpha(float _alpha) {
-    m_transform.state.alpha = CLAMP(_alpha, 0.0, 1.0);
-    m_dirty = true;
+    float alpha = CLAMP(_alpha, 0.0, 1.0);
+    if (m_transform.state.alpha != alpha) {
```

Comment:
The case that is interesting here is when the value stays 0.0 or 1.0 which I would think works with this test.

This is not a case where numeric instability can blow things up, so bit equality should suffice. Wouldn't mind to change this to a test < then some epsilon (which is still hard to get right though, see e.g. http://floating-point-gui.de/errors/comparison/ ).